extensions/amp-lightbox-gallery/validator-amp-lightbox-gallery.protoascii

```diff
@@ -35,8 +35,5 @@ attr_lists: {
   attrs: {
     name: "lightbox-thumbnail-id"
     value_regex_casei: "^[a-z][a-z\\d_-]*"
-    trigger: {
```

Comment:
/cc @Gregable 
This makes the validation much more permissive than it should be, but allows all legitimate use cases to work. Everything that should be disallowed is currently covered by runtime validation instead, in the form of `user().assert` logs. I can revert now / revisit when we have a good approach to validate this.